bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/unknown.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="shelly"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="shellydevice">
+		<label>Password protected Shelly device</label>
```

Comment:
```suggestion
		<label>Shelly Device</label>
```

Reply:
changed to:
		<label>Shelly Device</label>
		<description>A password protected or unknown device.</description>